lib/Cake/Test/Case/View/Helper/FormHelperTest.php

```diff
@@ -10648,4 +10648,42 @@ public function testInputDefaults() {
 		$this->assertEquals($expected, $result);
 	}
 
+/**
+ * Tests `_lastAction`.
+ *
+ *  With named, numeric value
+ *
+ * @return void
+ */
+	public function testLastActionWithNamedNumeric() {
+		$here = '/users/index/page:1';
+
+		$this->Form->request->here = $here;
+		$this->Form->create('User');
+
+		$expected = $here;
+		$actual = $this->Form->_lastAction;
```

Comment:
That's what I was thinking too, but it does work :-) Do you want me to use the PHPUnit assert?